pypsa_to_IPCC.py

```diff
@@ -9,26 +9,28 @@
 import pandas as pd
 
 #original IPCC file, official template
-path = "global_sectoral/global_sectoral/IPCC_AR6_WG3_Global_sectoral_Pathways_scenario_template_v3.1_20221027.xlsx"
+template_path = "global_sectoral/IPCC_AR6_WG3_Global_sectoral_Pathways_scenario_template_v3.1_20221027.xlsx"
 #use the official template
 
+# Metadata
 model = "PyPSA-Eur-Sec 0.0.2"
+model_version = "0.0.6"
+literature_reference = "Pedersen, T. T., Gøtske, E. K., Dvorak, A., Andresen, G. B., & Victoria, M. (2022). Long-term implications of reduced gas imports on the decarbonization of the European energy system. Joule, 6(7), 1566-1580."
+climate_target = 21 # CO2 budget
```

Comment:
should we write in the output file the intended emissions (i.e. the CO2 constraint) or the actual emissions (those read from the network file)?